addons/website_event_track/views/event_track_templates_agenda.xml

```diff
@@ -59,20 +72,7 @@
 
 <!-- Agenda Main Display -->
 <template id="agenda_main" name="Tracks: Main Display">
-    <!-- No tracks -->
-    <div class="col-12" t-if="not tracks_by_days">
-        <div class="h2 mb-3">No track found.</div>
-        <div t-if="search_key" class="alert alert-info text-center">
-            <p class="m-0">We did not find any track matching your <strong t-esc="search_key"/> search.</p>
-        </div>
-        <div t-else="" class="alert alert-info text-center" groups="event.group_event_user">
-            <a target="_blank" t-att-href="'/web?#action=website_event_track.action_event_track_from_event&amp;active_id=%s' % event.id">
-                <p class="m-0">Schedule some tracks to get started</p>
-            </a>
-        </div>
-    </div>
-
-    <section t-else="" class="col-12" t-foreach="days" t-as="day">
+    <section t-if="tracks_by_days" class="col-12" t-foreach="days" t-as="day">
```

Comment:
As anyway it is called by parent template only when having tracks_by_days, I would remove this if. That way it is now the caller's job to ensure it has values to display before calling this template.